src/Tribe/Date_Utils.php

```diff
@@ -99,7 +126,7 @@ public static function datepicker_formats( $translate = null ) {
 				return $formats;
 			}
 
-			return isset( $formats[ $translate ] ) ? $formats[ $translate ] : $formats[1];
+			return isset( $formats[ $translate ] ) ? $formats[ $translate ] : $formats[ static::get_datepicker_format_index() ];
```

Comment:
Is there any fault tolerance for the result of this method? What happens if the format isn't in the `$formats` array?

Reply:
No, not really. But That isn't a part of this hotfix. This is just replacing the number 1 with a static variable.

Reply:
✅